docs/features/techdocs/creating-and-publishing.md

```diff
@@ -30,7 +30,7 @@ scratch.
 Your working Backstage instance should by default have a documentation template
 added. If not, follow these
 [instructions](../software-templates/installation.md#adding-templates) to add
-the documentation template.
+the documentation template. The template creates a component with only TechDocs configuration as below mentioned in manual documentation setup, and is otherwise empty.
```

Comment:
```suggestion
the documentation template. The template creates a component with only TechDocs configuration and default markdown files as below mentioned in manual documentation setup, and is otherwise empty.
```
maybe worth do add that its not only configuration but also markdown files? what do you think?

Reply:
sounds good